AstarteDeviceSDKCSharp/Device/AstarteDevice.cs

```diff
@@ -251,6 +298,11 @@ public void AddInterface(string astarteInterfaceObject)
 
         }
 
+        /// <summary>
+        /// Return interface by interfaceName
+        /// </summary>
+        /// <param name="interfaceName">Name of specific interface</param>
+        /// <returns></returns>
```

Comment:
```suggestion
        /// <summary>
        /// Getter function for an interface.
        /// </summary>
        /// <param name="interfaceName">Name of specific interface</param>
        /// <returns>AstarteInterface with matching name when present, null otherwise.</returns>
```